heron/statemgrs/src/python/statemanager.py

```diff
@@ -122,7 +122,7 @@ def establish_ssh_tunnel(self):
     return localport
 
   def terminate_ssh_tunnel(self):
-    if self.tunnel:
+    if hasattr(self, 'tunnel'):
```

Comment:
Good question. Since `self.tunnel` is only set at [this line](https://github.com/twitter/heron/blob/e26c37a0fa04d6e39ff639753463db7ffbf638ce/heron/statemgrs/src/python/statemanager.py#L120) and I tried a meaningless command to open a subprocess, the `Popen` still gave me back an object instead of a `None`, I think not checking `None` should be fine here.